packages/datadog-plugin-pg/src/index.js

```diff
@@ -36,8 +38,14 @@ function createWrapQuery (tracer, config) {
 
       const originalCallback = pgQuery.callback
       const statement = pgQuery.text
+      const values = pgQuery.values
 
       span.setTag('resource.name', statement)
+      span.setTag('sql.query', statement)
```

Comment:
This should be removed for now because any obfuscation is done in the agent and only on the resource name, so any sensitive data from a regular query that is not a prepared statement would leak.

Reply:
Ok. I copied this from oracle plugin. Will remove it.

Reply:
Sounds like a bug in the `oracledb` plugin 😅.